services/spanishUtils.js

```diff
@@ -0,0 +1,186 @@
+/*
+Utilities to supplement conjugation library.
+*/
+
+
+/*shows what a verb would look like if it wasn't irregular. wraps the conjugation logic from the conjuation library 
+but decouples from irregularity.
+*/
+
+const conjugate = require('./conjugation');
+
+var verb_endings = [
+    "ar", "er", "ir",
+    "arse", "erse", "irse",
+    "ár", "ér", "ír",
+    "árse", "érse", "írse"
+]
+
+var ir_endings = {
```

Comment:
@sdee there is a way to share variables just like sharing functions from a library using exports in nodejs. However, here since these are indeed data, I think best to put them in json files like the others and require them into both files.